src/styles/index.scss

```diff
@@ -445,7 +445,7 @@ input[type='email']:focus {
 	#mc_embed_signup {
 		background: #fff;
 		clear: left;
-		font: 14px Helvetica, Arial, sans-serif;
+		font: 24px Helvetica, Arial, sans-serif;
```

Comment:
This seems excessively large, since the font size is populating for the form labels as well as the form title. If you want to update the title font size, I'd recommend creating additional CSS classes and adding `className="mc-field-label"` and `className="mc-field-error"` as appropriate.

CSS:
```css
    .mc-field-label {
        font-size: 20px;
    }

    .mc-field-error {
        margin-top: 10px;
        font-size: 16px;
    }
```

HTML/JSX:
```html
<label htmlFor="mce-FNAME" className="mc-field-label">
... ...
<ErrorMessage name="FNAME" component="div" className="mc-field-error"/>
```

## Current revision:

![form-labels-errors](https://user-images.githubusercontent.com/29528527/67647943-4a5e1f80-f8f1-11e9-91d3-0a64db6f6bb1.png)

## Suggested revision:

![form-labels-errors-suggestion](https://user-images.githubusercontent.com/29528527/67648181-3830b100-f8f2-11e9-91a0-afa050de0ac4.png)